tests/unit_tests/test_client.py

```diff
@@ -581,7 +582,37 @@ def test_uc_doesnt_count_metrics_for_dependency_parents(unleash_client):
     request = json.loads(responses.calls[-1].request.body)
     assert request["bucket"]["toggles"][child]["yes"] == 2
     assert request["bucket"]["toggles"][child]["variants"]["childVariant"] == 1
-    assert "Parent" not in request["bucket"]["toggles"]
+    assert parent not in request["bucket"]["toggles"]
+
+
+@responses.activate
+def test_uc_counts_metrics_for_child_even_if_parent_is_disabled(unleash_client):
+    # Set up API
+    responses.add(responses.POST, URL + REGISTER_URL, json={}, status=202)
+    responses.add(
+        responses.GET,
+        URL + FEATURES_URL,
+        json=MOCK_FEATURE_WITH_DEPENDENCIES_RESPONSE,
+        status=200,
+    )
+    responses.add(responses.POST, URL + METRICS_URL, json={}, status=202)
+
+    # Create Unleash client and check initial load
+    unleash_client.initialize_client()
+    time.sleep(1)
+
+    child = "WithDisabledDependency"
+    parent = "Disabled"
+    # Check a flag that depends on a disabled parent
+    unleash_client.is_enabled(child)
+    unleash_client.get_variant(child)
+
+    # Verify that the parent doesn't have any metrics registered
+    time.sleep(12)
```

Comment:
I think this is my only objection here. These tests are very painful to run since they all trigger long sleeps. This one adds a full 13 seconds

The client is being newed up with a 10 second metrics send interval. Can we see what happens to the test suite if we reduce that to 1 and trim the sleep down to say 2 seconds?

Reply:
Yeah, sure. I'll have a look. Think I might have had a look before, but can't remember why I didn't follow through in that case. Will check.

Reply:
And of course, we can move it out into a separate PR too. But just wanted some input on it first.

Reply:
> Yeah, sure. I'll have a look. Think I might have had a look before, but can't remember why I didn't follow through in that case. Will check.

I think I have too and failed to improve this a few times. Looks like the build server hates that change. I think these are accidentally doing more than just pausing execution. 

Okay, let's not do that then, this looks to be bigger than just this PR, I just to make sure it wasn't an easy fix we weren't doing.

Let's remove the commit, I'm okay for this to go in without the timing change

Reply:
Summary of in-person chat: I think I have a way to fix this. Will take those changes into a new PR (#305 tentatively)